include/TPP/Dialect/Xsmm/XsmmOps.td

```diff
@@ -304,4 +258,27 @@ def Xsmm_BrgemmDispatchOp : Xsmm_GemmLikeOp<"brgemm.dispatch"> {
   let hasVerifier = 1;
 }
 
+//===----------------------------------------------------------------------===//
+// FusedBrgemmDispatchOp
+//===----------------------------------------------------------------------===//
+
+def Xsmm_FusedBrgemmDispatchOp : Xsmm_Op<"fused_brgemm.dispatch", [Pure]> {
+  let summary = "dispatch fused brgemm operation.";
```

Comment:
Would be nice to add a description of which libxsmm function this will be calling and what the arguments mean.